scripts/serverless.js

```diff
@@ -481,7 +481,7 @@ processSpanPromise = (async () => {
     if (!isHelpRequest && (isInteractiveSetup || isStandaloneCommand)) {
       if (configuration) require('../lib/cli/ensure-supported-command')(configuration);
       if (isInteractiveSetup) {
-        if (!process.stdin.isTTY && !process.env.SLS_INTERACTIVE_SETUP_ENABLE) {
+        if (!require('@serverless/utils/log').isInteractive) {
```

Comment:
I've went with direct require here instead of top level as we already have a variable `isInteractiveSetup` that is very similar and could cause confusion